### ISSUE TITLE
Create a script checking that telegram bot works

### ISSUE THREAD

**Issue by aguschin** [aguschin/art-guide#50]
To monitor telegram bot health, we need an external way to make sure we get notified when something is broken.

As a first step, I suggest creating a test script that will take some images from reference database, send them to the telegram bot and check that the responses are correct. This should check whether:
- telegram bot is online
- we get right answers for images that are compressed by telegram, cropped by our cropping module, and searched by our reverse-image-search module

Basically, this is a simple end-to-end test that can be enhanced later: like send an alert to us (a message in telegram) stating that the bot is down. To interact with out bot from Python runtime, we need to use something like https://docs.telethon.dev/en/stable/basic/quick-start.html

**Comment by dinma-daniel** [comment]
Basic script checker created, waiting for approval to be merged in main. Currently on bot-status branch.

**Comment by aguschin** [comment]
Script was created, but we were unable to run it with GH actions since some authorization is required.
